sdk/messaging/azwebpubsub/options.go

```diff
@@ -0,0 +1,185 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator. DO NOT EDIT.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package azwebpubsub
+
+// ClientAddConnectionToGroupOptions contains the optional parameters for the Client.AddConnectionToGroup method.
+type ClientAddConnectionToGroupOptions struct {
```

Comment:
Now that the `HealthClient` is gone we can remove the `Client` prefix. Please take a look at how we do this in [`azkeys`](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/security/keyvault/azkeys/autorest.md), just search for "# delete client name prefix from method options and response types".

Reply:
Thanks for the suggestions, looks pretty neat.
Updated, as well as the APIview https://apiview.dev/Assemblies/Review/09874ac16a7f4fbaac2fe02b5af818cc